src/main/java/org/dsmhack/controller/LoginController.java

```diff
@@ -34,6 +34,9 @@
     @PostMapping("/login/sendCode")
     public ResponseEntity login(@RequestBody String emailAddress) {
         User user = userRepository.findByEmail(emailAddress);
+        if (user == null) {
```

Comment:
Optionals might shine in situations like this. More personal preference than anything else.

Reply:
I'm not familiar with Optionals, I'll see if I can look it up a little later.